packages/neuron-ui/package.json

```diff
@@ -65,7 +65,8 @@
     "react-i18next": "12.1.5",
     "react-router-dom": "6.14.0",
     "react-transition-group": "4.4.5",
-    "sass": "1.63.6"
+    "sass": "1.63.6",
+    "tslib": "2.6.2"
```

Comment:
why add something new to the bump PR

Reply:
Package at https://github.com/nervosnetwork/neuron/pull/2980#issuecomment-1855600816 throws the following error

```
[2023-12-19T06:09:38.627Z] [error] (0 , tslib_1.__spreadArray) is not a function
```

Not sure if it's a bug of babel, but it can be fixed by introduce `tslib` explicitly.